### DISCUSSION TITLE
How can I set the `command + c` action?

### DISCUSSION THREAD

**Discussion by flavioCap** [linearmouse/linearmouse#859] [Q&A]
Hi everyone, I wanted to ask you for help on configuring the keys on my mouse. I would like the side key to do `command + c`, to be able to perform the copy action. When I try to enter this command in the keyboard shortcuts section, only the `command` key symbol appears and not `command + c`. Thank you.

**Comment by lujjjh** [comment]
<img width="419" alt="image" src="https://github.com/user-attachments/assets/0d65271f-f0a7-4d07-a326-adf4f7a90c90" />

It works on my end. Do you have any clipboard apps installed that might have overridden the <kbd>Command</kbd>+<kbd>C</kbd> shortcut?

An alternative way is to manually edit the configuration file using a text editor rather than on the GUI.

**Reply by cypris75** [reply]
Have the same issue: 

<img width="1670" height="1208" alt="image" src="https://github.com/user-attachments/assets/3b50d040-a12c-4e86-ada4-ce0458222efc" />

Pressed Cmd + C but only the command options shows up. Using MacOS 27.0 on a Macbook Pro. Happens both on the external keyboard as well on the internal keyboard.

I also cannot just press "C" in recording mode. Does not do anything.

Config file:

```
{
  "$schema" : "https:\/\/schema.linearmouse.app\/0.11.4",
  "schemes" : [
    {
      "buttons" : {
        "mappings" : [
          {
            "action" : {
              "keyPress" : [
                "command"
              ]
            },
            "button" : 3,
            "command" : false,
            "control" : false,
            "option" : false,
            "shift" : false
          },
          {
            "action" : {
              "keyPress" : [
                "command"
              ]
            },
            "button" : 4,
            "command" : false,
            "control" : false,
            "option" : false,
            "shift" : false
          }
        ]
      },
      "if" : {
        "device" : {
          "category" : "mouse",
          "productID" : "0x1007",
          "productName" : "Wired Gaming Mouse",
          "vendorID" : "0x258a"
        }
      }
    }
  ]
}
```

**Reply by cypris75** [reply]
Fixed it manually via the `linearmouse.json` file like this:

```json
{
  "$schema": "https:\/\/schema.linearmouse.app\/0.11.4",
  "schemes": [
    {
      "buttons": {
        "mappings": [
          {
            "action": {
              "keyPress": [
                "command",
                "c"
              ]
            },
            "button": 3,
            "command": false,
            "control": false,
            "option": false,
            "shift": false
          },
...
```
